release/karaf/itests/switchyard/pom.xml

```diff
@@ -300,4 +301,35 @@
             </plugin>
         </plugins>
     </build>
+
+  <profiles>
+        <profile>
+            <id>fuse-6.2.1</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <!-- 2.18.1 is failing with "When writing xml report stdout/stderr: /tmp/stdout (No such 
+                            file or directory)" -->
+                        <version>2.16</version>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <java.awt.headless>true</java.awt.headless>
+                                <drools.karaf.features.classifier>${drools.karaf.features.classifier}</drools.karaf.features.classifier>
+                            </systemPropertyVariables>
+                            <includes>
+                                <include>**/*Test*.java</include>
+                            </includes>
+                            <excludes>
+                                <exclude>**/Abstract*.java</exclude>
+                                <exclude>**/HelpdeskDemoQuickstartTest.java</exclude>
+                                <exclude>**/*$*</exclude>
+                            </excludes>
+                        </configuration>
+                    </plugin>
```

Comment:
I think it is worth re-using as much as possible here. Otherwise it is going to be pain to maintain. I believe that at least the plugin version and the systemPropertyVariables could be removed as they will be taken from the default config.